appeal/service.go

```diff
@@ -129,25 +129,27 @@ func (s *Service) Create(appeals []*domain.Appeal) error {
 			activeAppeals[a.AccountID][a.ResourceID] != nil &&
 			activeAppeals[a.AccountID][a.ResourceID][a.Role] != nil {
 
-			if p.Config.Appeal.AllowActiveAccessExtensionIn == "" {
-				return ErrAppealFoundActiveAccess
-			}
+			if p.Config.Appeal != nil {
```

Comment:
@rahmatrhd what is the expected behaiour if `p.Config.Appeal` is nil

Reply:
guardian will ignore the appeal config, because rules like `AllowActiveAccessExtensionIn` and `AllowPermanentAccess ` are not present